developer/src/kmcmpdll/Compiler.cpp

```diff
@@ -2670,29 +2681,29 @@ DWORD process_if_synonym(DWORD dwSystemID, PFILE_KEYBOARD fk, LPWSTR q, LPWSTR t
   }
 
   tstr[(*mx)++] = UC_SENTINEL;
-  tstr[(*mx)++] = (WCHAR)CODE_IFSYSTEMSTORE;
-  tstr[(*mx)++] = (WCHAR)(dwSystemID + 1);   // I4785
+  tstr[(*mx)++] = (KMX_WCHAR)CODE_IFSYSTEMSTORE;
+  tstr[(*mx)++] = (KMX_WCHAR)(dwSystemID + 1);   // I4785
   tstr[(*mx)++] = 2;
-  tstr[(*mx)++] = (WCHAR)(dwStoreID + 1);
+  tstr[(*mx)++] = (KMX_WCHAR)(dwStoreID + 1);
   tstr[(*mx)] = 0;
 
   delete[] temp;
 
   return CERR_None;
 }
 
-DWORD process_if(PFILE_KEYBOARD fk, LPWSTR q, LPWSTR tstr, int *mx)  // I3431
+KMX_DWORD process_if(PFILE_KEYBOARD fk, LPKMX_WCHAR q, LPKMX_WCHAR tstr, int *mx)  // I3431
 {
   /* if(<store> <'='|'!='> <XString+outs>) */
-  DWORD i, code, not = FALSE;
-  LPWSTR r = q, s = q;
-  while (*s && *s != L' ' && *s != L'!' && *s != L'=') s++;
+  KMX_DWORD i, code; KMX_DWORD nnot = FALSE;
```

Comment:
```suggestion
  KMX_DWORD i, code;
  KMX_DWORD nnot = FALSE;
```

Whye `nnot` ?

Reply:
using "not" gives me:  error C2059: syntax error